cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/export/elements/GeneratingUnitEq.java

```diff
@@ -45,7 +45,7 @@ public static void write(String id, String generatingUnitName, EnergySource ener
         }
         if (energySource == EnergySource.WIND) {
             writer.writeEmptyElement(cimNamespace, "WindGeneratingUnit.windGenUnitType");
-            writer.writeAttribute(RDF_NAMESPACE, CgmesNames.RESOURCE, String.format("%s%s", cimNamespace, "WindGenUnitKind.onshore")); // all generators are considered onshore
+            writer.writeAttribute(RDF_NAMESPACE, CgmesNames.RESOURCE, String.format("%s%s", cimNamespace, "WindGenUnitKind." + windGenUnitType));
```

Comment:
Little suggestion, to be cleaner:
```suggestion
            writer.writeAttribute(RDF_NAMESPACE, CgmesNames.RESOURCE, String.format("%s%s%s", cimNamespace, "WindGenUnitKind.", windGenUnitType));
```

Reply:
Indeed, thanks